### DISCUSSION TITLE
Your Relationship With Paddle Needs Improvement

### DISCUSSION THREAD

**Discussion by debeer** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1210] [General]
Paddle is not doing a good job for you. I purchased a license for Al Dante Pro but have not had an email to acknowledge receipt. I was concerned that I had mistyped my email address and was able to confirm that I had done exactly that, when I started down the path with Paddle to resolve my problem. I t became obvious that I had typed .co instead of .com when entering my email. I contacted Paddle using the correct email address. Bizzarrely, their acknowledgement was directed to a completely different email address that I had used to buy a another product from a different vendor more than two years ago. Things fell apart after trying to talk to the paddlebot. On both occasions, after a lengthy scripted and futile interaction, the bot offered to refer the matter to (I assume) a human being, and even requested I provide an email address where I wanted to be contacted, which I provided.

Nothing so far.

It is unsatisfactory to have no direct way of contacting a developer in  a case like this. I know this was my own fault and I am willing to jump through hoops to fix the problem I caused - but there are no hoops to jump through in this case.

It is also not good practice to force resolution of what is not a payment or licensing problem through a third party - my contractual relationship is with AppHouseKitchen online because that is where I placed the order and that is where the order was acknowledged.

All I need  is a way for someone in authority to match my credit card payment to the incorrect email address, make the email correction, invalidate the licence sent to the wrong email address and issue a new licence.

Perhaps Paddle can be persuaded to actually talk to me so I can help them do a better job for you? I can't be the only idiot who excitedly mistypes his email address when ordering a neat new product!

**Comment by MatthiasKerbl** [comment]
Hi @debeer,

I am sorry to hear you had such a bad experience.

Usually, Paddle is taking care of typos like that without any issues. In some edge cases, they forward the request to us and we take care of it. However, this can take a couple of days depending on holidays/weekends/demand and so on.
Anyway, please let me know if you do not get an answer within a couple of days and I will also report your bad experience to Paddle.

Best,
Matthias
